infrastructure/templates/public-api/main.bicep

```diff
@@ -1,290 +1,206 @@
-param dbStorageSizeGB int = 32
-
-@description('Database : Azure Database for PostgreSQL Autogrow setting')
-param dbAutoGrowStatus string = 'Disabled'
-
-//Container Registry Params ----------------------------------------------------------------
-@minLength(5)
-@maxLength(50)
-@description('Registry : Name of the azure container registry (must be globally unique)')
-param containerRegistryName string
+param postgreSqlSkuName string = 'Standard_B1ms'
 
-@description('Deploy the Container Registry if you are not using an existing registry')
-param deployRegistry bool
+@description('Database : Azure Database for PostgreSQL Storage Size in GB.')
+param postgreSqlStorageSizeGB int = 32
 
-//Container App Params -------------------------------------------------------------------
-@minLength(2)
-@maxLength(32)
-@description('Specifies the name of the container app.')
-param containerAppName string
+@description('Database : Azure Database for PostgreSQL Autogrow setting.')
+param postgreSqlAutoGrowStatus string = 'Disabled'
 
-@description('Specifies the name of the container app environment.')
-param containerAppEnvName string = 'publicapi'
+@description('Database : Firewall rules.')
+param postgreSqlFirewallRules array = []
 
-@description('Specifies the name of the log analytics workspace.')
-param containerAppLogAnalyticsName string = 'publicapi'
+@description('Container App : Select if you want to use a public dummy image to start the container app.')
+param useDummyImage bool = true
 
-@description('Container App : Specifies the container image to deploy from the registry <name>:<tag>.')
-param containerAppImageName string
+param now string = utcNow('u')
 
-@description('Specifies the container port.')
-param containerAppTargetPort int = 80
-
-@description('Select if you want to use a public dummy image to start the container app.')
-param useDummyImage bool
-
-
-//ServiceBus Queue Params -------------------------------------------------------------------
-@description('Name of the Service Bus namespace')
-param namespaceName string = 'processor'
-
-@description('Name of the Queue')
-param queueName string = 'Processorqueue'
+//---------------------------------------------------------------------------------------------------------------
+// Variables and created data.
+//---------------------------------------------------------------------------------------------------------------
+var project = 'ees-publicapi'
+var resourcePrefix = '${subscription}-${project}'
+var storageAccountFullName = '${subscription}saeescore'
+var keyVaultName = 'kv-ees-01'
+var keyVaultFullName = '${subscription}-${keyVaultName}'
+var containerAppName = 'api'
+var containerAppImageName = useDummyImage ? 'azuredocs/aci-helloworld' : 'real-container-image-name'
+var containerAppTargetPort = 80
+var rootFileShareFolderName = 'data'
+var containerRegistryName = 'eesacr'
+var databaseNames = ['publicapi']
```

Comment:
We've called this `public_data` in the local Docker Compose setup. This is mostly to be a bit more consistent with how we've named the context i.e. `PublicDataDbContext`.

Might need a little chat, but we should probably try and be consistent in our naming regardless

Reply:
I've updated to "public_data" for now, but this'll likely be a throwaway database for now anyway, as I'll want to explore which other options we have for creating these new databases in the Bicep templates further on down the line.